Extensions/tweaks.js

```diff
@@ -1,5 +1,5 @@
 //* TITLE Tweaks **//
-//* VERSION 6.0.6 **/
+//* VERSION 6.0.7 **/
```

Comment:
needs rebase/version rebump

```suggestion
//* VERSION 6.0.8 **/
```

Reply:
hm wait no it doesnt as long as you merge this before your other Tweaks PR